packages/caravan-clients/src/client.test.ts

```diff
@@ -479,6 +479,7 @@ describe("BlockchainClient", () => {
       // Verify the returned result
       expect(result.utxos).toEqual(
         await Promise.all(
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Can you not use your newly exported `UTXO` interface from `packages/caravan-clients/src/client.ts`?

Reply:
I tried that and forget why I reversed it. It might’ve just been I was trying to clean up other lint issues in parallel and the scope was increasing. I’ll try again here to at least make this unnecessary. Thanks!

Reply:
[994068e](https://github.com/caravan-bitcoin/caravan/pull/16/commits/994068e300a2404a9f8fd169b78d2973606af278)